kubernetes/linux/mdsd.xml

```diff
@@ -48,20 +48,31 @@
   -->
   <Schemas>
     <Schema name="ContainerLogSchema">
-		<Column name="Id" type="str" mdstype="mt:wstr" />
-		<Column name="LogEntry" type="str" mdstype="mt:wstr" />
-		<Column name="LogEntrySource" type="str" mdstype="mt:wstr" />
-		<Column name="LogEntryTimeStamp" type="str" mdstype="mt:wstr" />
-		<Column name="TimeOfCommand" type="str-rfc3339" mdstype="mt:utc" />
-		<Column name="SourceSystem" type="str" mdstype="mt:wstr" />
-		<Column name="Computer" type="str" mdstype="mt:wstr" />
-		<Column name="Image" type="str" mdstype="mt:wstr" />
-		<Column name="Name" type="str" mdstype="mt:wstr" />
-  </Schema>
+      <Column name="Id" type="str" mdstype="mt:wstr" />
+      <Column name="LogEntry" type="str" mdstype="mt:wstr" />
+      <Column name="LogEntrySource" type="str" mdstype="mt:wstr" />
+      <Column name="LogEntryTimeStamp" type="str" mdstype="mt:wstr" />
+      <Column name="TimeOfCommand" type="str-rfc3339" mdstype="mt:utc" />
+      <Column name="SourceSystem" type="str" mdstype="mt:wstr" />
+      <Column name="Computer" type="str" mdstype="mt:wstr" />
+      <Column name="Image" type="str" mdstype="mt:wstr" />
+      <Column name="Name" type="str" mdstype="mt:wstr" />
+    </Schema>
+    <Schema name="ContainerLogV2Schema">
+      <Column name="ContainerId" type="str" mdstype="mt:wstr" />
+      <Column name="LogMessage" type="str" mdstype="mt:wstr" />
+      <Column name="LogSource" type="str" mdstype="mt:wstr" />
+      <Column name="TimeGenerated" type="str" mdstype="mt:wstr" />
+      <Column name="PodName" type="str-rfc3339" mdstype="mt:utc" />
```

Comment:
there is no type checking (thats why data is flowing) as everything is json string to ODS now. BUT, this has to be a string (and not rfc3339 like timeofcommand), as time of command is guaranteed to be in rfc3339 format, since we convert it unlike timegenerated. So i will be making them string.